ussologin/ussologin.go

```diff
@@ -32,9 +31,12 @@ var (
 	otpKey  = "Two-factor auth (Enter for none)"
 )
 
-// Login completes the login information using the provided filler
-// and attempts to obtain a USSO token using this information.
-func Login(filler form.Filler) (*usso.SSOData, error) {
+// GetToken uses filler to interact with the user and uses the provided
+// information to obtain an OAuth token from Ubuntu SSO. The returned
+// token can subsequently be used with LoginWithToken to perform a login.
+// The tokenName argument is used as the name of the generated token in
```

Comment:
Can tokenName be anything? As someone not that familiar with OAuth, this doesn't help me decide what value to pass here.